### PR TITLE
The ASP.NET Core 6 Web API a Three-Tier Architecture.

### LINKED ISSUE #6728: The ASP.NET Core 6 Web API a Three-Tier Architecture.
#### Proposed title of article
The ASP.NET Core 6 Web API a three-tier architecture.

#### Proposed article introduction

Web APIs with.NET Core and ASP.NET Core ASP.NET Core is a new web framework created on top of .NET Core by Microsoft to replace legacy technology that dates back to .NET 1.0. Three-tier architecture is a well-known software application architecture that divides application forms into three physical and logical computing tiers: the presentation tier, or user interface; the application tier, in which the data is processed; and the data tier, where the application's information is stored and handled.

We'll look at three-tier architecture in this tutorial, as well as how to include the Data Access Layer and Business Access Layer into a project and how they interact.

#### Key takeaways
- Overview on what is  The ASP.NET Core 6 Web API and the three-tier architecture.
- Project Structure
  1. Presentation Layer (PL)
  2. Business Access Layer (BAL)
  3. Data Access Layer (DAL)
- Add the References of the Project
- Data Access Layer
- Business Access Layer
- Presentation layer
- Advantages and dis-advantages of 3-Tier Architecture

#### Article quality
This article addresses the ASP.NET Core 6 Web API and the three-tier architecture in detail. The objective is to guarantee that the topic is easy to grasp for a beginning while simultaneously providing a reference piece for intermediate and advanced users to refresh their memory from an introductory standpoint.

#### References
N/A

### PR DESCRIPTION
**Pull request by njorosirmie**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes

`This closes #6728

### CODE REVIEW

**Comment by njorosirmie** [comment]
@hectorkambow kindly assign this article for review. Thanks!

**Comment by njorosirmie** [comment]
@hectorkambow kindly assign this. Thanks.

**Comment by fely545** [comment]
@hectorkambow If allowed I would like to review this article has been in the queue for a long time
Cc @louisefindlay23 @ahmadmardeni1

**Comment by ahmadmardeni1** [comment]
Hey! @fely545 Are you a peer reviewer?

**Comment by WanjaMIKE** [comment]
Plagiarism check done

**Comment by hectorkambow** [comment]
@section-engineering-education/content-moderator-team  

Lets be sure not to overlook this PR.

File: content/authors/samson-njoroge/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+title: Samson Njoroge
```
Comment (Jethro-magaji) [inline content/authors/samson-njoroge/index.md:2]: This section isn't arranged in the proper Markdown format. Check out this [example](https://github.com/section-engineering-education/engineering-education/blob/master/content/authors/ahmad-mardeni/index.md) to fix it.

File: content/articles/the-asp-net-core-6-web-api-a-three-tier-architecture/index.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
+### Introduction
+This article aims at making a view of how to include layers in a project. The layers discussed are the middle and Data Access layers. The article will also discuss the applications of these layers in detail.
+
+A three-tier architecture aims at dividing and administering applications into three analytical and physical computing layers. ASP.NET core Web API is an interface medium for web servers and applications. A core 6 Web design consists of improvements that have notably decreased the number of lines and codes sufficient in the design of an application.
+
+### Prerequisites
```
Comment (Jethro-magaji) [inline content/articles/the-asp-net-core-6-web-api-a-three-tier-architecture/index.md:6]: This section should come after the "Table of contents" section.

**Review by Jethro-magaji — CHANGES_REQUESTED** [review]
Hi! @njorosirmie 
Well done for such an amazing article. However, there are some corrections you need to make.
Run your article on tools like Grammarly and ProWritingAid for better editing, this would save the time to review your work.
Please, replace all the images you have with good quality/clear images and use a high-resolution free stock image for your hero image. 
Here's a [link](https://github.com/section-engineering-education/engineering-education/pull/6210/commits/acc44893d4ec258f4e077b9ce8e3e2ca2b9d6a9f) to a PR with little or no revision to guide you on formatting. However, if you have any questions or need help, you can reach out to me on the EngEd Slack channel.
Thanks.

**Comment by hectorkambow** [comment]
Closing this PR due to it not being ready to be published 👍

**Pull request closed without merging**